code/kg2/extract-chembl.sh

```diff
@@ -9,36 +9,36 @@ if [[ "${1:-}" == "--help" || "${1:-}" == "-h" ]]; then
     exit 2
 fi
 
-MYSQL_DBNAME=${1:-"chembl"}
+mysql_dbname=${1:-"chembl"}
 
 echo "================= starting build-chembl.sh ================="
 date
 
-CONFIG_DIR=`dirname "$0"`
-source ${CONFIG_DIR}/master-config.shinc
+config_dir=`dirname "$0"`
+source ${config_dir}/master-config.shinc
 
-CHEMBL_DIR=${BUILD_DIR}/chembl
-CHEMBL_VERSION=25
-CHEMBL_DB_TARBALL=chembl_${CHEMBL_VERSION}_mysql.tar.gz
-CHEMBL_SQL_FILE=${CHEMBL_DIR}/chembl_${CHEMBL_VERSION}/chembl_${CHEMBL_VERSION}_mysql/chembl_${CHEMBL_VERSION}_mysql.dmp
+chembl_dir=${BUILD_DIR}/chembl
+chembl_version=25
+chembl_db_tarball=chembl_${chembl_version}_mysql.tar.gz
+chembl_sql_file=${chemble_dir}/chembl_${chembl_version}/chembl_${chembl_version}_mysql/chembl_${chembl_version}_mysql.dmp
 
-rm -r -f ${CHEMBL_DIR}
-mkdir -p ${CHEMBL_DIR}
+rm -r-f ${chemble_dir}
```

Comment:
ope - looks like a couple typos here?

- `-r -f` was changed to `-r-f`
- looks like you have an extra 'e' in '`chemble_dir`'